lib/rake/javaextensiontask.rb

```diff
@@ -211,11 +211,14 @@ def java_classpath_arg(*args)
         end
       end
       unless jruby_cpath
-        jruby_cpath = ENV['JRUBY_PARENT_CLASSPATH'] || ENV['JRUBY_HOME'] &&
-          Dir.glob("#{File.expand_path(ENV['JRUBY_HOME'])}/lib/*.jar").
-            join(File::PATH_SEPARATOR)
```

Comment:
No, it's not like that. `rbconfig` is a replacement of `JRUBY_HOME`. So it will work even if the `JRUBY_HOME` is not defined and pick the lib path from `RbConfig::Config`.

Reply:
For java8 or earlier, `java.class.path` or `sun.boot.class.path` will set it. So, there's no worry.